src/store/dicom.js

```diff
@@ -149,14 +149,11 @@ export default (dependencies) => ({
       const updatedSeriesKeys = []; // to be returned to caller
 
       await Promise.all(
-        updatedSeries.map(async (gdcmSeriesUID) => {
-          const numberOfSlices = await dicomIO.buildSeriesOrder(gdcmSeriesUID);
-          let seriesKey;
-
-          if (gdcmSeriesUID in state.seriesIndex) {
-            seriesKey = state.seriesIndex[gdcmSeriesUID].SeriesInstanceUID;
-          } else {
-            const info = await dicomIO.readSeriesTags(gdcmSeriesUID, [
+        updatedSeries.map(async (seriesKey) => {
+          const numberOfSlices = await dicomIO.buildSeriesOrder(seriesKey);
+
+          if (!(seriesKey in state.seriesIndex)) {
+            const info = await dicomIO.readSeriesTags(seriesKey, [
```

Comment:
Is the `gdcmSeriesUID in state.seriesIndex` scenario not handled anymore?

Reply:
This will be irrelevant once we refactor to reference volumeIDs rather than seriesUIDs.